src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -435,6 +435,30 @@ def test_storage_account_with_shared_key_access(self, resource_group):
         self.cmd('az storage account update -n {} --allow-shared-key-access true'.format(name),
                  checks=[JMESPathCheck('allowSharedKeyAccess', True)])
 
+    @api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2021-02-01')
+    @ResourceGroupPreparer(location='eastus', name_prefix='cli_storage_account')
+    def test_storage_account_with_key_and_sas_policy(self, resource_group):
```

Comment:
could you add test with 100000 days because it is a workaround to disable as e-mail mentioned?

Reply:
Sure